test/tests/functional/pbs_release_limited_res_suspend.py

```diff
@@ -1059,3 +1059,56 @@ def test_server_restart_with_suspened_job(self):
         self.server.sigjob(jobid=jid1, signal="resume")
         self.server.expect(JOB, {ATTR_state: 'R'}, id=jid1)
         self.server.expect(NODE, 'jobs', op=SET, id=self.mom.shortname)
+
+    def test_server_restart_with_suspened_job_unset(self):
+        """
+        Test that when the attribute is set and unset,
+        the server does not crash on restart with a suspended job.
+        """
+        # Set ncpus in restrict_res_to_release_on_suspend server attribute
```

Comment:
All the comments in this test explain what the code does.  The code does that.  Either explain why you are doing something or remove them.